api/MediaDevices.json

```diff
@@ -225,6 +225,75 @@
           }
         }
       },
+      "getDisplayMedia": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getDisplayMedia",
+          "description": "<code>getDisplayMedia()</code>",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "72"
+              },
+              {
+                "version_added": "70",
+                "version_removed": "72",
+                "alternative_name": "navigator.getDisplayMedia",
+                "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
```

Comment:
Being in 72 Canary is no guarantee that it will be shipping in 72. Can you please sit on this for a few weeks?

Reply:
These should all be fixed in the changes I just submitted.